src/bitmessagekivy/mpybit.py

```diff
@@ -1,34 +1,198 @@
+# pylint: disable=no-name-in-module, too-few-public-methods, import-error, unused-argument
+# pylint: disable=attribute-defined-outside-init
+
 """
-    Dummy implementation for kivy Desktop and android(mobile) interface
+Bitmessage android(mobile) interface
 """
-# pylint: disable=too-few-public-methods
 
-from kivy.app import App
-from kivy.uix.label import Label
+import os
+import json
+import importlib
+
+from kivy.lang import Builder
+from kivy.lang import Observable
+from kivy.clock import Clock
+from kivy.properties import (
+    BooleanProperty,
+    NumericProperty,
+    StringProperty,
+    ListProperty
+)
+from kivy.metrics import dp
+from kivy.uix.boxlayout import BoxLayout
+from kivy.uix.spinner import Spinner
+from kivy.core.window import Window
+
+
+from kivymd.app import MDApp
+from kivymd.uix.list import (
+    OneLineAvatarIconListItem,
+    OneLineListItem
+)
+from kivymd.uix.bottomsheet import MDCustomBottomSheet
+
+from debug import logger
 
 from pybitmessage.bitmessagekivy.kivy_state import KivyStateVariables
+from pybitmessage.bmconfigparser import config
+
+with open(os.path.join(os.path.dirname(__file__), "screens_data.json")) as read_file:
```

Comment:
maybe add a test that `screens_data.json` is a valid json? Then it would make sense to create a function to load it, and a test to test that function.